packages/nuxt3-module/src/index.ts

```diff
@@ -11,25 +11,27 @@ export default defineNuxtModule<ShopwareNuxtOptions>({
     configKey: "shopware",
   },
   async setup(moduleConfig, nuxt) {
+    const shopwareEndpoint =
+      moduleConfig.shopwareEndpoint ?? "https://demo-frontends.shopware.store";
+    const accessToken =
+      moduleConfig.shopwareAccessToken ?? "SWSCBHFSNTVMAWNZDNFKSHLAYW";
+
     addPluginTemplate({
       filename: "runtime/shopware.plugin.mjs",
       // eslint-disable-next-line @typescript-eslint/ban-ts-comment
       // @ts-ignore
       src: resolve(__dirname, "../plugin.ts"),
       options: {
-        shopwareEndpoint:
-          moduleConfig.shopwareEndpoint ??
-          "https://demo-frontends.shopware.store",
-        shopwareAccessToken:
-          moduleConfig.shopwareAccessToken ?? "SWSCBHFSNTVMAWNZDNFKSHLAYW",
+        shopwareEndpoint: shopwareEndpoint,
+        shopwareAccessToken: accessToken,
         shopwareApiClient: {
           timeout: moduleConfig.apiClientConfig?.timeout ?? "10000",
         },
       },
     });
 
     // TODO: remove it once nitro server build contains all external packages of nuxt3-module (composables-next)
-    nuxt.options.build.transpile.push("@shopware-pwa/composables-next");
+    //nuxt.options.build.transpile.push("@shopware-pwa/composables-next");
```

Comment:
shall we remove it completely?

Reply:
I just commented out to see if some test or something is failing in the preview.
But currently, I do not see any problem with removing it completely.

So remove it?

Reply:
I'm not sure what the purpose was, but maybe to tell nuxt that `composables-next` package must not be imported by nuxt end project (because it's a part of nuxt3-module bundle?). @patzick please help here 😄

Reply:
yeah it looks like was needed on early stages, but if that's working now we can remove it or clean afterwards after we release it